src/pages/jobfound/components/ListFound.tsx

```
@@ -52,88 +47,50 @@ const ListFound = ({ page, setTotalPages }: ListFoundProps) => {
-              />
-
-              <div className="mt-[14px] w-[360px]">
-                <div className="flex flex-row text-purple-500 font-B02-SB">
-                  {item.requiredCertification},{item.workTimeInfo},
-                  {item.physicalInfo}
-                </div>
-                <div
-                  className="mt-[6px] text-gray-900 font-T04-SB"
+            <div className="mb-4 flex items-center justify-center rounded-[30px] border border-gray-300 bg-white pb-[30px]">
+              <div
+                key={item.jobId}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)